seed/seed.json

```diff
@@ -1114,6 +1114,33 @@
                 "channel": ["NIGHTLY"],
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
+        },
+        {
+            "name": "BraveDebounceStudy",
+            "experiments": [
+                {
+                    "name": "Enabled",
+                    "probability_weight": 5,
+                    "feature_association": {
+                        "enable_feature": ["BraveDebounce"]
+                    }
+                },
+                {
+                    "name": "Disabled",
+                    "probability_weight": 95,
+                    "feature_association": {
+                        "disable_feature": ["BraveDebounce"]
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "channel": ["NIGHTLY"],
+                "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
+            }
```

Comment:
maybe add min version, analogous to e.g. https://github.com/brave/brave-variations/pull/125/files#diff-7ac2945ede68c73330363911678f6c9a2a6290b70e2dd170868295f65da7e872R1113

Reply:
added in https://github.com/brave/brave-variations/commit/8a08e3cb52f8f5b37f3b3f6908dad9fd1d794593